doc/src/manual/faq.md

```diff
@@ -29,7 +29,7 @@ you can redefine types and constants.  You can't import the type names into `Mai
 to be able to redefine them there, but you can use the module name to resolve the scope.  In other
 words, while developing you might use a workflow something like this:
 
-```
+```julia
```

Comment:
This PR is the last for the manual. Might go through the devdocs or something next, but nothing will conflict. I don't mean to rush, I just want to try to direct discussion back to the matter at hand. 😉